CONTRIBUTING.md

```diff
@@ -213,6 +214,21 @@ In the worst case, trying to make as few modifications as possible:
 6. Make a PR of your changes.
 
 
+Commit guidelines
+-----------------
+- All commits should have a subject and a body
+- Wrap commit subjects and bodies to 72 characters
+- Sign-off your commits
+- Add a best-effort scope designation to commit subjects. This could be a directory name, file name,
+  or the name of a logical grouping of code. Examples:
+  - **[dir]** ceph-releases: change flavor specification(s) in the `ceph-container` dir
+  - **[dir]** jewel: change flavor spec for jewel
+  - **[dir]** kubernetes: edit a Kubernetes example in `examples/kubernetes`
+  - **[file]** osd_disk_activate: edit the `src/daemon/osd_scenarios/osd_disk_activate.sh` file
+  - **[logical group]** osd prep: change how OSDs are prepared, with changes in multiple files
+  - **[combinations]** jewel osd prep: change how OSDs are prepared only in jewel
+
+
```

Comment:
Commit message should 
- describe the problem addressed by this commit
- explain the chosen solution